### DISCUSSION TITLE
Dexcom G7 Setup

### DISCUSSION THREAD

**Discussion by CJWTX** [gui-dos/DiaBLE#12] [General]
Would love to set this up and try it out. I have the app installed from testflight, but I'm having trouble getting it setup to read from my sensor. I took a look around the github and didn't find much documentation. Any help would be greatly appreciated.

**Comment by gui-dos** [comment]
After selecting “Dexcom” in the Settings as your preferred transmitter and switching to the third “Test” mode in the Console, you should be able to receive the same data that the Dexcom app is getting running in parallel.

As you can see in the screenshots I posted commenting <https://github.com/gui-dos/DiaBLE/commit/ce85f28>, DiaBLE can parse the current 186 glucose value and the exact warmup time which is 27 minutes and not the “30 minutes” as the marketing department simplifies.

As they say, the “devil is in the details…” ;-)


![image](https://github.com/gui-dos/DiaBLE/assets/7220550/3f25a438-15a0-476d-a46c-e6bd530f7f45)

![image](https://github.com/gui-dos/DiaBLE/assets/7220550/88b79d82-2275-4e77-88d7-71835ec6e901)
